astropy/samp/hub.py

```diff
@@ -943,14 +1046,22 @@ def _notify(self, private_key, recipient_id, message):
         self._update_last_activity_time(private_key)
 
         if private_key in self._private_keys:
-            if self._is_subscribed(self._public_id_to_private_key(recipient_id),
-                                   message["samp.mtype"]) is False:
-                raise SAMPProxyError(2, "Client {} not subscribed to MType {}"
-                                    .format(recipient_id, message["samp.mtype"]))
-
-            self._launch_thread(target=self._notify_, args=(private_key,
-                                                            recipient_id,
-                                                            message))
+            if (
+                self._is_subscribed(
+                    self._public_id_to_private_key(recipient_id), message["samp.mtype"]
+                )
+                is False
```

Comment:
We should probably change this to ``not ...``

Reply:
Changed in commit 716d079. Feel free to revert it.